### PR TITLE
Changes to get supercloud results

### PR DESCRIPTION
**Pull request by nkumar-bdai**
Results from this:
cleanup_playroom_num_online_transitions_perc_solved

### CODE REVIEW

File: predicators/ground_truth_models/spot_env/options.py [hunk 1]
```diff
@@ -494,7 +494,8 @@ def _place_object_on_top_policy(state: State, memory: Dict,
     # Special case: the robot is already on top of the surface (because it is
     # probably the floor). When this happens, just drop the object.
     surface_geom = object_to_top_down_geom(surface_obj, state)
-    if surface_geom.contains_point(robot_pose.x, robot_pose.y):
+    if surface_geom.contains_point(
+            robot_pose.x, robot_pose.y) and surface_obj.name == "floor":
```
Comment (tsilver-bdai) [inline predicators/ground_truth_models/spot_env/options.py:498]: what happened here? just curious

File: predicators/ml_models.py [hunk 2]
```diff
@@ -1378,6 +1378,7 @@ def get_option(self,
                    epsilon: float = 0.0) -> _Option:
         """Get the best option under Q, epsilon-greedy."""
         # Return a random option.
+        # import ipdb; ipdb.set_trace()
```
Comment (tsilver-bdai) [inline predicators/ml_models.py:1381]: remove

**Review by tsilver-bdai — APPROVED** [review]
excellent work!

File: predicators/ground_truth_models/spot_env/options.py (hunk 1 above)
Comment (nkumar-bdai) [reply predicators/ground_truth_models/spot_env/options.py:498]: oh - sometimes it appears the robot gets on top of the table in simulation?

**Pull request merged**